rds/rds.tf

```diff
@@ -1,9 +1,12 @@
 locals {
   major_engine_version      = split(".", var.engine_version)[0]
   final_snapshot_identifier = "final-snapshot-${var.project}-${var.environment}-${formatdate("DD-MM-YY-hhmm", timestamp())}"
+  # @NOTE: `"default.postgres${local.major_engine_version}"` does not exist for e.g. postgres15 eventhough, postges 15 is generally available
```

Comment:
Can we skip the creations of the 13/14 parameter group if the engine_version is not 13/14?